lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.ts

```diff
@@ -99,6 +99,8 @@ export class StartProcessCloudComponent implements OnChanges, OnInit {
                 this.processPayloadCloud.processDefinitionKey = null;
                 if (this.processDefinition.valid) {
                     this.setProcessDefinitionOnForm(processDefinitionName);
```

Comment:
There's a memory leak in this component, it does not unsubscribe from service. That means each time this component is displayed - it is multiplying in memory and listening to events

Reply:
@SilviuCPopa you can check this example on how to setup ngOnDestroy and "takeUntil" pipe to remove memory leak: https://github.com/Alfresco/alfresco-ng2-components/blob/ac224a0b82cfa3894e83cb186341f53b6eaf6bbc/demo-shell/src/app/components/files/files.component.ts#L274